### PR TITLE
1640/fix/fix patient display age

### PR DESCRIPTION
**Pull request by chenFriedman**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx [hunk 1]
```diff
@@ -113,7 +113,7 @@ const InvestigatedPersonInfo = (props: Props) => {
         }
         return status === InvestigationMainStatusCodes.NEW;
     };
-
+console.log(age, investigationStaticInfo)
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx:116]: Remove log

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx [hunk 2]
```diff
@@ -267,7 +267,7 @@ const InvestigatedPersonInfo = (props: Props) => {
                         <>
                             <InfoItemWithIcon testId='age' name='גיל' value={+age < 1 ? 'פחות משנה' : age} icon={CakeOutlined} />
                             {
-                                +age <= maxComplexityAge && <ComplexityIcon tooltipText='המאומת מתחת לגיל 15' />
+                                +age <= maxComplexityAge && <ComplexityIcon tooltipText='המאומת מתחת לגיל 14' />
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx:270]: Isn't there an existing PR w/ this change?

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx (hunk 2 above)
Comment (chenFriedman) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx:270]: you're right, I got out of this branch by mistake

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx (hunk 1 above)
Comment (chenFriedman) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx]: fixed

**Review by Noam6421 — APPROVED** [review]

**Review by MagusM — APPROVED** [review]

File: client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx (hunk 2 above)
Comment (MagusM) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx]: is it resolved?

Comment (chenFriedman) [reply client/src/components/App/Content/InvestigationForm/InvestigationInfo/InvestigatedPersonInfo/InvestigatedPersonInfo.tsx]: yes uploaded to dev in different branch

**Review by y-golde — APPROVED** [review]

**Pull request merged**